RadixWallet/Features/TransactionReviewFeature/TransactionReview+Sections.swift

```diff
@@ -313,9 +316,12 @@ extension TransactionReview {
 
 	private func extractDapps(
 		componentAddresses: [ComponentAddress],
+		lockerAddresses: [LockerAddress],
```

Comment:
I think it should work if we have a single argument `addresses: [Address]`, and then `extractDappEntities` is reused.

Reply:
updated